### PR TITLE
Gh 489 prototyping with tabbar

### PR DESCRIPTION
**Pull request by sche**

### CODE REVIEW

File: Multisig/UI/App/MainTabView.swift [hunk 1]
```diff
@@ -9,58 +9,55 @@
...
-            SettingsView()
-                .gnoTabItem(id: ViewStateMode.settings) {
+            MainContentView(SettingsView())
+                .tabItem {
                     VStack {
                         Image("tab-icon-settings")
                         Text("Settings")
                     }
                 }
+                .tag(2)
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/MainTabView.swift]: Can we avoid using constants?we can use ViewStateMode underlying value for instance

File: Multisig/UI/UI Library/HideSystemNavigationBarModifier.swift [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 import SwiftUI
 
+#warning("TODO: do we need this modifier?")
```
Comment (MouazAlzahabi) [inline Multisig/UI/UI Library/HideSystemNavigationBarModifier.swift]: I also don't think we will use it

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/UI/App/MainTabView.swift (hunk 1 above)
Comment (sche) [reply Multisig/UI/App/MainTabView.swift]: ok

File: Multisig/UI/UI Library/HideSystemNavigationBarModifier.swift (hunk 2 above)
Comment (sche) [reply Multisig/UI/UI Library/HideSystemNavigationBarModifier.swift]: removed

**Pull request merged**